shepherd_bms/include/datastructs.h

```diff
@@ -1,5 +1,66 @@
 #ifndef DATASTRUCTS_H
 #define DATASTRUCTS_H
 
+#include <nerduino.h>
+
+
+
+/*Individual chip data*/
+typedef struct ChipData_t
+{
+    uint16_t VoltageReading[12];          //store voltage readings from each chip
+    uint16_t thermistorReading[32];       //store all therm readings from each chip
+    bool discharge[12];
```

Comment:
Are we doing cell resistances seperately as they will be over a longer time-scale?